public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/response_parser.ts

```diff
@@ -147,6 +147,27 @@ export default class ResponseParser {
   static dateTimeToEpoch(dateTimeValue: any) {
     return dateTime(dateTimeValue).valueOf();
   }
+
+  static parseSubscriptions(result: any): Array<{ text: string; value: string }> {
```

Comment:
I would avoid using `any`, if this is an API response, we should declare what structure we expect. For example, is `.data.value` always present?

Reply:
I copied it over as is from `azure-monitor-datasource`.

I created a separated work item clean this up:
#34860